_posts/2021-08-04-cockpit-250.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Cockpit 250
+author: pitti
+date: '2021-08-04'
+tags: cockpit, machines
+slug: cockpit-250
+category: release
+summary: Cockpit with kpatch and VM migration support and session menu improvements
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly.
+
+Here are the release notes from Cockpit 250 and cockpit-machines 249:
+
+
+## Shell: Improve admin switcher and session menu
+
+Cockpit has a "limited access" mode with lowered privileges, where browsing generally works, but changing things that require administration rights generally does not.
+
+If someone is stuck in "limited access" mode without knowing how to switch, it would appear that Cockpit is "broken" or cannot perform tasks it should be able to do. To make it more obvious how to switch modes, Cockpit now has several "eye-catching" changes to make switching easier to notice. These changes are in the shell, which is visible on every page within Cockpit, and the overview page's alert.
+
+- On the overview alert, a warning color has been used to draw attention
+- A lock icon has been used in both the overview alert and the shell privilege escalation icon
+- The shell's escalation action now resembles a button
+
+![panel-or8]({{ site.baseurl }}/images/250-shell-improve-admin-switcher-and-session-menu.png)
+
+All of the above changes only affect "limited access" mode. Normal administration mode looks the same as before.
+
+The menu on the top right is now called "Session" instead of showing an user icon. This clarifies that the settings in that menu apply to remote hosts as well, which possibly run as a different user.
+
+*Thanks to Verhoeckx for discussing and suggesting these design changes!*
```

Comment:
Possibly link to his account?

Reply:
Done, and added black borders to the other two images.